src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/jpa/entity/RecommendationEntity.kt

```diff
@@ -37,7 +42,7 @@ enum class Recommendation(val text: String) {
 }
 
 enum class Status {
-  DRAFT
+  DRAFT, SUBMITTED
```

Comment:
How about `DOCUMENT_CREATED` rather than `SUBMITTED` (which is quite generic and might be read as 'submitted to delius' or 'submitted to PPCS' which could both be future statuses?)

Reply:
I don't think 'submitted to Delius' would ever be a state, considering Delius will one day be deprecated. I'd envisage this being a flag on the case if it was needed, especially if the document needed to be submitted to multiple places - you wouldn't want to store this in the status as they'd end up writing over each other. I tried not to put too much thought into this as it kind of depends on frontend design and don't think we've got that far yet. Let me speak to Beck and see what she thinks.

Reply:
Updated to `DOCUMENT_CREATED` after discussing with Beck